omnibus/config/software/python3.rb

```diff
@@ -16,8 +16,8 @@
     source :sha256 => "b7c36f7ed8f7143b2c46153b7332db2227669f583ea0cce753facf549d1a4239"
   end
```

Comment:
Can we get rid this old `3.6.7` version definition at the same time? AFAIK it's not used since it's overwritten by the following `3.7.4` version definition.

Reply:
done